c++/src/capnp/reconnect.c++

```diff
@@ -120,6 +134,11 @@ private:
       return result;
     }
 
+    AnyPointer::Pipeline sendForPipeline() override {
+      // TODO(bug): This definitely fails to detect disconnects; see comment in send().
```

Comment:
I felt like it's more important to be consistent with `send()`, so that people can safely change `send()` to `sendForPipeline()` without having to ask the question "Is it at all possible that I might be operating on a reconnect capability here?"